tt_metal/llrt/tt_memory.h

```diff
@@ -38,8 +38,9 @@ class memory {
 
  public:
   memory();
-  memory(std::istream& is);
+  memory(std::string const &path);
 
+ public:
```

Comment:
extraneous public?

Reply:
sorry, I missed this -- just focussed on the approval.  While not necessary, having an access specifier before each segment of a class defn makes for more readable code, IMHO